fusioncrafter.lua

```diff
@@ -125,16 +125,11 @@ local function craft(recipe)
 
   -- Put input items into their places
   for item, amount in pairs(recipe.input) do
-
-    -- If item is the core item transfer it to the core inventory
-    if item == recipe.core then
-      transfer(item, amount, input, core)
-    else  -- Else put it into the injectors
-      transfer(item, amount, input, injectors)
-    end
-
+    transfer(item, amount, input, injectors)
   end
 
+  transfer(recipe.core, amount, input, core)
```

Comment:
Thank you very much for this great contribution! :smile:  

I tested it on ATM3 Remix and got an error and found a little mistake. Now that the `recipe.core` is a table that maps item id to amount, you'd also need to iterate over the items. For example you could do it like this:
```suggestion
  -- Transfer items to core
  for item, amount in pairs(recipe.core) do
    transfer(item, amount, input, core)
  end
```

Otherwise great fix for my somewhat bodged code. :sweat_smile: